File: Lab_C#/Asignacion_02(Do_While)/Asignacion_02(Do_While).cs
using System;

class Program
{
    static void Main(string[] args)
    {   //variables para almacenar los meses, sus temperaturas, la suma total y el promedio
        int meses = 1;
        int temperatura;
        int total_temp = 0;
        int promedio_temp;

    do 
    {
        Console.Write ("Ingresar la temperatura del mes " + meses + ": "); //solicitar al usuario que introduzca la temperatura
        temperatura = int.Parse (Console.ReadLine());
    
        total_temp = total_temp + temperatura; //almacenamiento de temperaturas previamente registradas
        meses = meses + 1; //incrementa el numero de meses en 1 hasta alcanzar el mes 12
    }
    while (meses <= 12); //bucle While que repite mientras meses sea menor o igual a 12

    promedio_temp = total_temp / 12; //promedio de temperaturas dividiendo las sumas entre los meses

    Console.Write ("El promedio anual de la temperatura es: " + promedio_temp + ": ");
    }
}

File: Lab_C++/Modulo_5/Asignacion_02(Do_While).cpp
#include <iostream>
using namespace std;

int main() 
{
    int meses = 1;
    int temperatura;
    int total_temp = 0;
    int promedio_temp;

    do 
    {
        cout << "Introduce el promedio del mes " << meses << ": ";
        cin >> temperatura;
    
        total_temp = total_temp + temperatura;
        meses = meses + 1;
    }
    while (meses <= 12);

    promedio_temp = total_temp / 12;

    cout << "El promedio anual de temperatura es: " << promedio_temp << endl;

    return 0;
}